tests/acceptance/common.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# Copyright 2016 Mender Software AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from fabric.api import *
+import fabric.network
+
+import pytest
+import os
+import signal
+import subprocess
+import time
+
+import conftest
+
+# Return Popen object
+def start_qemu():
+    # Relies on being run from a meta-layer next to meta-mender-qemu.
+    proc = subprocess.Popen("../../../meta-mender-qemu/scripts/mender-qemu")
+    execute(qemu_prep_after_boot, hosts = conftest.current_hosts())
+    return proc
+
+
+def kill_qemu():
+    os.system("pkill qemu-system-arm")
```

Comment:
I know, I don't like this one either. I tried to use `subprocess.terminate()`, but I couldn't get this to work, despite trying a lot. It seemed surprisingly difficult to do it the right way, so in the end I settled for this. I agree we should change this in the more longterm, but I think it's not worth spending more time on it right now.